src/routes/(admin)/profile/+page.server.ts

```
@@ -0,0 +1,80 @@
+	save: async ({ request, fetch, cookies }) => {
+		const token = getSessionTokenCookie();
+
+		const form = await superValidate(request, zod(editUserSchema));
+		if (!form.valid) {
+			return fail(400, {
+				form
+			});
+		}
+
+		let res: Response;
+		const updateData: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)